tests/spec/fs.link.spec.js

```diff
@@ -111,3 +111,119 @@ describe('fs.link', function() {
     });
   });
 });
+
+
+
+
+
+describe('fs.promises.link', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
+    expect(fs.promises.link).to.be.a('function');
+  });
+
+  it('should create a link to an existing file', function(done) {
+    var fs = util.fs();
+
+    fs.open('/myfile', 'w+', function(error, fd) {
+      if(error) throw error;
+
+      fs.close(fd, function(error) {
+        if(error) throw error;
+
+        fs.promises.link('/myfile', '/myotherfile', function(error) {
```

Comment:
Should follow the promise syntax
```js
fs.promises.link('/myfile', '/myotherfile')
	.then()..
```

Reply:
Done for tests:  
- create a link to an existing file
- not follow symbolic links 

although I could not get it running for not allowing links to a directory